settings.py

```diff
@@ -22,3 +22,25 @@
 to be at least greater than FLOW_STATS and ideally it slightly greater than
 whichever longest network convergence FlowMods operations that your network has.
 """
+
+# Rate limits for sending flow mods this can be set per NApp,
+# and the pacing is per DPID when in the context of that NApp
+# The NApp that the flow pertains to is determined via the `owner` attribute of the flow.
+ACTION_PACES = {
+    "send_flow_mod.no_owner": {
+        "pace": "100/second",
+        "strategy": "fixed_window",
+    },
+    "send_flow_mod.mef_eline": {
+        "pace": "100/second",
+        "strategy": "fixed_window",
+    },
+    "send_flow_mod.of_multi_table": {
+        "pace": "100/second",
+        "strategy": "fixed_window",
+    },
+    "send_flow_mod.telemetry_int": {
+        "pace": "100/second",
```

Comment:
This rate is reasonable although it's lower than what we currently have. There's still an issue to tune it https://github.com/kytos-ng/telemetry_int/issues/40, so it's OK. I'll set that issue for `2024.1` and then in a next stress test that I do with telemetry_int I'll check this out.